test/test_variant.py

```diff
@@ -41,6 +41,9 @@ def test_insertion_shared_prefix():
     eq_(variant.start, 11)
     eq_(variant.end, 11)
     eq_(variant.short_description, "chr1 g.11_12insT")
+    assert variant.is_indel
```

Comment:
I like eq_ because it comes with a default message of the wrong value but with booleans, what does `ok_` buy us?